src/index.ts

```diff
@@ -69,6 +69,22 @@ export function getFormattedSrcLink(filepath: string, repoUrl?: RepoUrl) {
   return srcLink
 }
 
+export function prepareTodosForDanger(keywords: string[] | undefined, addedText: string, removedText: string, filepath: string, repoUrl: string | ((filepath: string) => string) | undefined | boolean, keywordMatches: PlainObject<string[]>) {
```

Comment:
```suggestion
export function prepareTodosForDanger(keywords: string[] | undefined, addedText: string, removedText: string, filepath: string, repoUrl: RepoUrl, keywordMatches: PlainObject<string[]>) {
```

Can use the exported type above